sdk/communication/Azure.Communication.Identity/src/CommunicationIdentityClient.cs

```diff
@@ -82,7 +82,7 @@ private CommunicationIdentityClient(string endpoint, HttpPipeline httpPipeline,
         {
             _clientDiagnostics = new ClientDiagnostics(options);
             RestClient = new CommunicationIdentityRestClient(_clientDiagnostics, httpPipeline, endpoint, options.ApiVersion);
-            TurnRestClient = new CommunicationIdentityRestClient(_clientDiagnostics, httpPipeline, endpoint, "2021-12-12");
+            TurnRestClient = new CommunicationIdentityRestClient(_clientDiagnostics, httpPipeline, endpoint, "2021-02-22-preview1");
```

Comment:
Unfortunately the change to unify both Identity and TURN clients didn't make it into the PROD deployment, so I couldn't fix this hacky version of using two separate clients here.

Reply:
When does it reach prod?

Reply:
It reached Prod yesterday